### DISCUSSION TITLE
guests in presta are not converting to customers - stats

### DISCUSSION THREAD

**Discussion by adamus007p** [PrestaShop/PrestaShop#37649] [Ideas]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

guests in presta are not converting to customers - stats
guests are seperated and you can not track it, what was before it came a customer

there is a lot of guests in database which is useless, just a database is bigger and bigger

### Steps to reproduce

go to shop, add some product, then register as customer and check the database to analize it

### PrestaShop version(s) where the bug happened

8.2

**Comment by Hlavtox** [comment]
That is how prestashop works. Guests are separated entity.

**Comment by paulnoelcholot** [comment]
@PrestaShop/product-council What did you think about that?

Thanks

**Comment by adamus007p** [comment]
Is there a sense of it? For me not, it not give anything if you will not covert a guest to a customer. Keeping that all data in database it is a waste of resources.
